features/step_definitions/skip_to_main_link_steps.rb

```diff
@@ -0,0 +1,23 @@
+When(/^I visit the homepage$/) do
+  home_page.load
+end
+
+When(/^I view an article$/) do
+  browse_to_article article('en')
+end
+
+When(/^I visit a category page$/) do
+  browse_to_category(category, 'en')
+end
+
+Then(/^I should be able to skip to the main content$/) do
+  expect(page).to have_selector('div#main')
```

Comment:
@johnplayer1982 not sure about this mate, these three steps are all just checking that there is a `main#main` element on the page, regardless of where.

What I had in mind when we were chatting before was something along the lines of adding `t-*` selectors to the relevant elements, then using those to see if they were the same elements as `main#main`.

Something along the lines of:

```
expected_element = page.find('.t-default-main-content')
main_element = page.find('main#main')
expect(main_element).to eq expected_element
```